src/cf-forms.less

```diff
@@ -35,12 +38,17 @@
 // .error
 @input-error:                   #ea130b;
 
-// warning
+// .warning
 @input-warning:                 #ffb108;
 
 // .success
 @input-success:                 #009d38;
 
+// range input
+@input-range-bg:                #329d83;
+@input-range-border:            #009d88;
+@input-range-handle:            #dddd38;
```

Comment:
I think we should probably have separate variables for the track border and the handle border. I suggest:

```less
@input-range-track-bg
@input-range-track-border
@input-range-handle-bg
@input-range-handle-border
```